Gemfile

```diff
@@ -45,6 +45,7 @@ group :development, :test do
   gem 'orderly'
   gem 'simplecov'
   gem 'launchy'
+  gem 'solargraph'
```

Comment:
do we want to keep this in the gem file or was this for demonstration purposes only?